### ISSUE TITLE
L.Draw.Circle - leaflet-1.0 compatibility

### ISSUE THREAD

**Issue by JrFolk** [Leaflet/Leaflet.draw#480]
Leaflet-1.0-beta only provides legacy support on the API for `L.circle`. call but Leaflet.draw, as well as many other plugins, uses the capitalized constructor rather than the API. 

[I made a PR to address the issue](https://github.com/Leaflet/Leaflet/pull/4290) but in the meantime I have tested [a workaround](https://github.com/JrFolk/Leaflet.draw/commit/df82f503f52f0487354444d4eb301615551069c7) for the current stable release (Leaflet 0.7.7) as well as the 1.0-beta/dev branch.

**Comment by ddproxy** [comment]
Does this equate to a change in L.Draw?

**Comment by JrFolk** [comment]
Yes, [this workaround for L.Draw](https://github.com/JrFolk/Leaflet.draw/commit/df82f503f52f0487354444d4eb301615551069c7) is tested and ready to go but luckily, my PR to Leaflet was merged promptly and is now live in the [**Leaflet 1.0-dev**](http://leafletjs.com/download.html) branch, so it's only affecting those using L.Draw with **Leaflet 1.0 beta 2**.  I've been using the fix with the current L.Draw release and it works just fine.  This is probably a non-issue at this point.

I just wanted to touch base with both projects because it caused me a lot of head scratches for the better part of a morning;  I wanted to save others the same headache ASAP.
